search/engine.go

```diff
@@ -426,7 +426,13 @@ func (e *ESEngine) DoSearch(ctx context.Context, query Query, sortBy string, fro
 	if err != nil {
 		return nil, errors.Wrap(err, "ESEngine.DoSearch - Error multisearch Do.")
 	}
-
+	hasSpanish := false
```

Comment:
Change this to shouldMergeResults and comment that we test this with Spanish for now.

Reply:
done